src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {person.name}</h2>
```

Comment:
Recommend destructuring the props, it makes the code cleaner and easier to read.

Example:
```
const { name, age, sex, isMarried, partnerName } = person;
```